README.md

```diff
@@ -182,19 +182,7 @@ channel.subscribe('myEvent', function (message) {
 
 Subscribing to a channel in delta mode enables [delta compression](https://www.ably.com/documentation/realtime/channels/channel-parameters/deltas). This is a way for a client to subscribe to a channel so that message payloads sent contain only the difference (ie the delta) between the present message and the previous message on the channel.
 
-Provide the library with Vcdiff decoding capability - see
-[Delta Plugin](#delta-plugin).
-
-Request a Vcdiff formatted delta stream using channel options when you get the channel:
-
-```js
-var channelOptions = {
-  params: {
-    delta: 'vcdiff',
-  },
-};
-var channel = client.channels.get('test', channelOptions);
-```
+Configuring a channel for deltas is detailed in the [@ably-forks/vcdiff-decoder documentation](https://github.com/ably-forks/vcdiff-decoder#usage).
 
 Beyond specifying channel options, the rest is transparent and requires no further changes to your application. The `message.data` instances that are delivered to your listening function continue to contain the values that were originally published.
```

Comment:
Unsure about leaving this information in here, or removing this section entirely, thoughts @owenpearson ?

Reply:
I don't have any strong opinions tbh, I wouldn't be against moving this to the vcdiff-decoder too but I also think it's fine to leave it here so I'm happy for you to make the final call.